sql/driver/conn.go

```diff
@@ -53,7 +55,18 @@ func (c *conn) Exec(stmt string, args []driver.Value) (driver.Result, error) {
 	if err != nil {
 		return nil, err
 	}
-	return driver.RowsAffected(len(result.Rows)), nil
+	switch t := result.GetUnion().(type) {
+	case nil:
+		return nil, nil
+	case *Response_Result_DDL_:
```

Comment:
Is there a reason to distinguish this from `Response_Result_RowsAffected`? I would think the latter with a value of 0 could cover this usage as well.